### DISCUSSION TITLE
error al crear item

### DISCUSSION THREAD

**Discussion by mendez19** [podio-community/podio-php#247] [Q&A]
estoy obteniendo esta respuesta al momento de crear un item 2024-02-19 21:48:22 400 POST /item/app/18274947/ 2024-02-19 21:48:22 Request body: {"fields":{"sucursalagente":[114],"nombre-del-cliente":"prueba nombre cajero 2","telefono":"(224)331-9470","guia":"Guia de prueba","tipo-de-servicio":[1],"status-de-guia":[2],"datos-de-la-guia":"Prueba 2","tipo":[3],"asunto":"prueba problematica","responsable-de-dar-solucion":[164579850]},"app":{"app_id":18274947}} 2024-02-19 21:48:22 Response: {"error":"field.app.invalid","error_detail":null,"error_description":"The referenced item is not valid","error_parameters":{},"error_propagate":true,"request":{"url":"http://api.podio.com/item/app/18274947/","method":"POST","query_string":""}}
Fatal error: Uncaught PodioBadRequestError: "The referenced item is not valid" Request URL: http://api.podio.com/item/app/18274947/ Stack Trace: #0 C:\xampp\htdocs\podio\vendor\podio-community\podio-php\lib\PodioClient.php(361): PodioClient->request('POST', '/item/app/18274...', Array, Array) #1 C:\xampp\htdocs\podio\vendor\podio-community\podio-php\models\PodioItem.php(186): PodioClient->post('/item/app/18274...', Array) #2 C:\xampp\htdocs\podio\vendor\podio-community\podio-php\models\PodioItem.php(74): PodioItem::create(Object(PodioClient), 18274947, Array, Array) #3 C:\xampp\htdocs\podio\prueba.php(38): PodioItem::save(Object(PodioClient), Object(PodioItem)) #4 {main} thrown in C:\xampp\htdocs\podio\vendor\podio-community\podio-php\lib\PodioClient.php on line 307

esta es la petición de creacion 	$fields = new PodioItemFieldCollection(array(
		  new PodioAppItemField(array("external_id" => "sucursalagente", "values" => array("item_id" => 114))),
		  new PodioTextItemField(array("external_id" => "nombre-del-cliente", "values" => "prueba nombre cajero 2")),
		  new PodioTextItemField(array("external_id" => "telefono", "values" => "(224)331-9470")),
		  new PodioTextItemField(array("external_id" => "guia", "values" => "Guia de prueba")),
		  new PodioCategoryItemField(array("external_id" => "tipo-de-servicio", "values" => 1)),
		  new PodioCategoryItemField(array("external_id" => "status-de-guia", "values" => 2)),
		  new PodioTextItemField(array("external_id" => "datos-de-la-guia", "values" => "Prueba 2")),
		  new PodioCategoryItemField(array("external_id" => "tipo", "values" => 3)),
		  new PodioTextItemField(array("external_id" => "asunto", "values" => "prueba problematica")),
		  new PodioContactItemField(array("external_id" => "responsable-de-dar-solucion", "values" => array("profile_id" => 164579850) ))
		));

**Comment by daniel-sc** [comment]
Are you sure the referenced item with id `114` is present in the referenced app? 114 seems like a wrong id - you can get the id of an item in the web frontend by selecting "Actions > Developer Info"
